hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/TestECContainerRecovery.java

```diff
@@ -94,6 +94,7 @@ public static void init() throws Exception {
     conf.setFromObject(clientConfig);
 
     conf.setTimeDuration(HDDS_SCM_WATCHER_TIMEOUT, 1000, TimeUnit.MILLISECONDS);
+    conf.set("hdds.scm.replication.thread.interval", "10s");
```

Comment:
To avoid raw use of config string:

```java
    ReplicationManager.ReplicationManagerConfiguration rmConfig =
        conf.getObject(ReplicationManager.ReplicationManagerConfiguration.class);
    rmConfig.setInterval(Duration.of(10, ChronoUnit.SECONDS));
    // there are other intervals such as setOverReplicatedInterval()
    // and setUnderReplicatedInterval(), but maybe not needed in this test.
    conf.setFromObject(rmConfig);
```

Reply:
done